src/identity/components/GlobalHeader/IdentityUserAvatar.tsx

```diff
@@ -0,0 +1,131 @@
+import React, {CSSProperties} from 'react'
+import {IdentityUser} from 'src/client/unityRoutes'
+import {
+  Button,
+  ButtonShape,
+  ClickOutside,
+  ComponentColor,
+  FlexBox,
+  Icon,
+  IconFont,
+  InfluxColors,
+} from '@influxdata/clockface'
+
+import './UserPopoverStyles.scss'
+import {Link} from 'react-router-dom'
+
+type Props = {
+  user: IdentityUser
+}
+
+type State = {
+  isPopoverOpen: boolean
+}
+
+class IdentityUserAvatar extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props)
+    this.state = {
+      isPopoverOpen: false,
+    }
+  }
+
+  private getInitials = (): string => {
+    const {user} = this.props
+    const firstName = user.firstName
+    const lastName = user.lastName
+    const initials = `${firstName.charAt(0)}${lastName.charAt(0)}`
+    return initials
+  }
+
+  // get button style based on whether popover is open or not
+  private getButtonStyle = (): CSSProperties => {
```

Comment:
This is a nice function, but I think it's doing the same thing as the [`classnames` ](https://www.npmjs.com/package/classnames) package we import elsewhere.  I think this is good as is, but just a thought.

Reply:
Fixed this, thank you for suggesting this. looks much cleaner as well.